presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/LogsHandler.java

```diff
@@ -31,6 +30,7 @@
 import java.time.format.DateTimeFormatterBuilder;
 import java.time.format.DateTimeParseException;
 import java.util.function.Predicate;
+import java.util.logging.Level;
```

Comment:
It doesn't have a constant representing "all levels", which makes it less useful for our purposes.